torch/csrc/jit/script/tree_views.h

```diff
@@ -587,12 +587,27 @@ struct Assign : public Stmt {
       const Expr& rhs) {
     return Assign(Compound::create(TK_ASSIGN, range, {lhs, rhs}));
   }
+  static Assign create(
```

Comment:
This should be `Assign(Expr lhs, Expr rhs, Maybe<Expr> type)` instead of two different constructors.

Also can you update the big comment at the top of the file with the new form?